### DISCUSSION TITLE
How to configure a FreeBSD or Ubuntu server as a client to connect a strongSwan roadwarrior service provider

### DISCUSSION THREAD

**Discussion by TomHsiung** [strongswan/strongswan#2028] [General]
For some reason, I would like to configure one of my servers as a roadwarrior client but I don't know how to. Is there any scenario? Thanks

**Comment by tobiasbrunner** [comment]
The [quickstart page](https://docs.strongswan.org/docs/5.9/config/quickstart.html) in the docs has roadwarrior examples.

**Reply by TomHsiung** [reply]
Got it, thanks. And is it compatible for a client with present `swanctl.conf` to connect with a strongSwan service which is based on `ipsec.conf`?

**Reply by tobiasbrunner** [reply]
Sure.

**Reply by TomHsiung** [reply]
Thanks and I will try later.
